packaging/test_irods_resource_plugin_s3.py

```diff
@@ -150,6 +150,9 @@ def test_put_get_small_file_in_repl_node(self):
     def test_put_get_large_file_in_repl_node(self):
         pass
 
+    @unittest.skipIf(True, 'test does not work in decoupled because we are using same bucket for multiple resources')
```

Comment:
`decoupled`?  or `detached`?

Reply:
In this case decoupled is correct.  These are tests where we use decoupled naming so the key is the reverse object id.  That breaks tests that use the same bucket for multiple resources.